source/agora/node/Ledger.d

```diff
@@ -542,17 +564,20 @@ unittest
 {
     import agora.common.crypto.Key;
 
+    auto gen_key_pair = getGenesisKeyPair();
```

Comment:
This kind of refactoring is good, but better done in its own commit

Reply:
Yes,